src/app/tabs/testTab/testTab.js

```diff
@@ -8,22 +8,27 @@ class TestTabLogic {
   }
 
   generateTestFile () {
-    var path = this.fileManager.currentPath()
-    var fileProvider = this.fileManager.fileProviderOf(path)
+    const path = this.fileManager.currentPath()
+    const fileName = this.fileManager.currentFile()
+    const fileProvider = this.fileManager.fileProviderOf(path)
     if (!fileProvider) return
-    helper.createNonClashingNameWithPrefix(path + '/test.sol', fileProvider, '_test', (error, newFile) => {
+    helper.createNonClashingNameWithPrefix(fileName, fileProvider, '_test', (error, newFile) => {
```

Comment:
might be that no file are selected, we need to default to something, or is the button disabled in that case?

Reply:
it is disabled

Reply:
button is disabled in that case